docs/tutorials/python/tutorial_scripts/dataset_collection.py

```diff
@@ -0,0 +1,75 @@
+"""Here is where you'll find the code for the DatasetCollection tutorial."""
+
+import pandas as pd
+
+from synapseclient import Synapse
+from synapseclient.models import Column, ColumnType, Dataset, DatasetCollection, Project
+
+# First, let's get the project that we want to create the DatasetCollection in
+syn = Synapse()
+syn.login()
+
+project = Project(name="My Testing Project").get()  # Replace with your project name
```

Comment:
Usually I'd been using `"My uniquely named project about Alzheimer's Disease"` to keep up the general theme of the tutorials:

```
Welcome to the Synapse Python client! In the following tutorials you'll create a new project and learn about things you can do with that project.

These examples are themed around a biomedical researcher working with Alzheimer's Disease wanting to share their findings with their colleagues, and the world.
```

https://synapsepythonclient--1189.org.readthedocs.build/en/1189/tutorials/python_client/